java/code/webapp/WEB-INF/decorators/layout_c.jsp

```diff
@@ -64,5 +64,22 @@
         </c:if>
       </script>
     </div>
+    <script>
```

Comment:
In general, we do not want to add stray global scripts like this, they don't show up in intellisense when you check scripts and they also do not have Typescript or other checks.